src/platform/imx8ulp/include/platform/lib/mailbox.h

```diff
@@ -0,0 +1,70 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Peng Zhang
+ */
+
+#ifdef __SOF_LIB_MAILBOX_H__
+
+#ifndef __PLATFORM_LIB_MAILBOX_H__
+#define __PLATFORM_LIB_MAILBOX_H__
+
+#include <sof/lib/memory.h>
+#include <stddef.h>
+#include <stdint.h>
+
+/*
+ * The Window Region on i.MX8 SRAM is organised like this :-
+ * +--------------------------------------------------------------------------+
+ * | Offset              | Region         |  Size                             |
+ * +---------------------+----------------+-----------------------------------+
+ * | SRAM_TRACE_BASE     | Trace Buffer   |  SRAM_TRACE_SIZE                  |
+ * +---------------------+----------------+-----------------------------------+
+ * | SRAM_DEBUG_BASE     | Debug data     |  SRAM_DEBUG_SIZE                  |
+ * +---------------------+----------------+-----------------------------------+
+ * | SRAM_INBOX_BASE     | Inbox          |  SRAM_INBOX_SIZE                  |
+ * +---------------------+----------------+-----------------------------------+
+ * | SRAM_OUTBOX_BASE    | Outbox         |  SRAM_MAILBOX_SIZE                |
+ * +---------------------+----------------+-----------------------------------+
+ */
+
+#define MAILBOX_DSPBOX_SIZE		SRAM_OUTBOX_SIZE
+#define MAILBOX_DSPBOX_BASE		SRAM_OUTBOX_BASE
+#define MAILBOX_DSPBOX_OFFSET		SRAM_OUTBOX_OFFSET
+
+#define MAILBOX_HOSTBOX_SIZE		SRAM_INBOX_SIZE
+#define MAILBOX_HOSTBOX_BASE		SRAM_INBOX_BASE
+#define MAILBOX_HOSTBOX_OFFSET		SRAM_INBOX_OFFSET
```

Comment:
Please try to align.

Reply:
Align is OK as far as i can tell. The github GUI makes it look wrong.